primitiv/c/define.h

```diff
@@ -36,4 +36,7 @@ typedef _Bool bool;
 #endif  // SWIG
```

Comment:
Lines 27-35:
- Use `#elif defined(_MSC_VER)` instead of `COMPILER_MSVC`.
- `PRIMITIV_COMPILE_LIBRARY` looks to be used in MSVC, are there any documents in this repository?